internal/sdk/cloudian/sdk_test.go

```diff
@@ -1,16 +1,62 @@
 package cloudian
 
 import (
+	"context"
 	"encoding/json"
 	"errors"
 	"fmt"
 	"math/rand"
+	"net/http"
+	"net/http/httptest"
 	"reflect"
 	"strings"
 	"testing"
 	"testing/quick"
 )
 
+func TestGetGroup(t *testing.T) {
+	expected := Group{
+		GroupID: "QA",
+	}
+
+	mockServer := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.WriteHeader(http.StatusOK)
+		w.Write([]byte(`{
+			"groupId": "QA"
+			}`))
```

Comment:
Just do a simple single line, or are we expecting a lot of fields in the future?

Reply:
Just my obsessive prefence for not prematurely increasing complexity before it's required 😅